style.css

```diff
@@ -584,6 +687,31 @@ textarea {
     border: 1px solid #8993a4;
   }
 
+  .project {
+    display: none;
+  }
+
+  .projdesk {
+    display: block;
+    width: 124px;
+    height: 48px;
+    margin-top: 15px;
+    background: #ff6b00;
+    color: #fff;
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 17px;
+    border: none;
+  }
+
+  .projdesk:hover {
+    transition: all 0.2s ease-in-out;
+    background: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
   .cards {
```

Comment:
- [ ] [REQUIRED] Good job so far! You need to create a `See Project`("In [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471), this button should appear on hover, otherwise it won't!") button for all cards in the desktop version, but in the desktop version, you only created it for the `2 cards` on the main `top` and the `first row`. Please use the `image` below to correct the style mistakes!

![image](https://user-images.githubusercontent.com/63915024/188952337-f3429955-3c7f-4104-8652-00691089df4f.png)

Reply:
- [ ] Just as the previous reviewer has earlier suggested, You need to create a "See Project" button for all your project cards in the desktop view which should appear when you hover over the project card itself. Kindly fix it 👍🏽